### PR TITLE
upd to 2.38

### PR DESCRIPTION
**Pull request by ekomarova**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ekomarova** [comment]
Forgot to update sha

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,12 +25,14 @@ requirements:
 
 test:
     commands:
-        - test -f ${PREFIX}/lib/libuuid.a  # [unix]
+        - test -f ${PREFIX}/lib/libuuid.a      # [unix]
+        - test -f ${PREFIX}/lib/libuuid.so     # [linux]
+        - test -f ${PREFIX}/lib/libuuid.dylib  # [osx]
```
Comment (ocefpaf) [inline recipe/meta.yaml:30]:
```suggestion
        - test -f ${PREFIX}/lib/libuuid.a  # [not win]
        - test -f ${PREFIX}/lib/libuuid${SHLIB_EXT}  # [not win]
```

**Comment by ekomarova** [comment]
@ocefpaf Coul you please help to resolve the license error?
`ValueError: License file given in about/license_file (/Users/runner/work/1/s/recipe/LICENSE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,12 +25,13 @@ requirements:
...
     commands:
-        - test -f ${PREFIX}/lib/libuuid.a  # [unix]
+        - test -f ${PREFIX}/lib/libuuid.a  # [not win]
+        - test -f ${PREFIX}/lib/libuuid${SHLIB_EXT}  # [not win]
 
 about:
     home: http://sourceforge.net/projects/libuuid/
     license: BSD-3-Clause
-    license_file: Documentation/licenses/COPYING.BSD-3
+    license_file: LICENSE.txt
```
Comment (ocefpaf) [inline recipe/meta.yaml:34]:
```suggestion
    license_file: COPYING
```

**Comment by ocefpaf** [comment]
Something is off here b/c osx_arm64 passed but the hook did not update it here. See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=681697&view=logs&j=ced5d8de-8227-5f3f-33a1-45cf1592c45a

Merging this. Thanks @ekomarova for the PR!

**Pull request merged**